PyTorch/Detection/SSD/main.py

```diff
@@ -123,24 +125,30 @@ def make_parser():
 def train(train_loop_func, logger, args):
     # Check that GPUs are actually available
     use_cuda = not args.no_cuda
+    train_samples = 118287
```

Comment:
Why is this hardcoded here ?

Reply:
It is hard-coded here: https://github.com/HerringForks/DeepLearningExamples/blob/master/PyTorch/Detection/SSD/src/data.py#L36 . 
- The training samples are fixed for COCO-2017 dataset so we can hard-code the value